src/Select/index.js

```diff
@@ -50,6 +52,45 @@ export const getItemByValue = (value, items) => {
   return founditem || "";
 };
 
+/**
+ * @param {Object} item an item from `items`
+ * @param {array} items downshift index `items`
+ * @returns {Number} index of item
+ */
+export const getItemIndex = ({ props }, items) => {
+  const values = items.map(({ props }) => props.value);
+  return values.indexOf(props.value);
+};
+
+/**
+ * @param {Object} highlightedIndex index of currently highlight item
+ * @param {Array} categoryChildren child items in a given category
+ * @param {Array} items downshift `items`
+ * @returns {Boolean} if the provided item is in the category
+ */
+export const isHighlightedInCategory = (
+  highlightedIndex,
+  categoryChildren,
+  items
+) => {
+  if (highlightedIndex < 0) return false;
+  const highlightedValue = items[highlightedIndex].props.value;
+  const categoryValues = categoryChildren.map((child) => child.props.value);
+  return categoryValues.includes(highlightedValue);
```

Comment:
Might be ok, but this makes things get a bit wonky when you have 2 items with the same value
<img width="918" alt="image" src="https://github.com/narmi/design_system/assets/51931432/cca5382d-f05b-4697-936a-ec28cdf7da3a">

Reply:
This was because I didn't have them have different values